docs/root/configuration/operations/overload_manager/overload_manager.rst

```diff
@@ -8,8 +8,9 @@ The :ref:`overload manager <arch_overview_overload_manager>` is configured in th
 field.
 
 An example configuration of the overload manager is shown below. It shows a
-configuration to drain HTTP/X connections when heap memory usage reaches 95%
-and to stop accepting requests when heap memory usage reaches 99%.
+configuration to drain HTTP/X connections when heap memory usage reaches 92%,
+to stop accepting requests when heap memory usage reaches 95% and to stop accepting
+new TCP connections when memory usage reaches 95%.
```

Comment:
nit: refer each item to its name in the configuration, and consider using different numbers for the stop-accepting-requests and stop-accepting-new-TCP-connections values.

Reply:
done referenced which configuration does what.